src/libs/actions/IOU.js

```diff
@@ -152,6 +152,32 @@ function rejectTransaction({
         });
 }
 
+/**
+ * Resets IOU currency to preferredCurrency
+ *
+ * @param {Object} params.selectedCurrencyCode
+ * @param {Object} params.selectedCurrencySymbol
+ */
+function setSelectedCurrency({selectedCurrencyCode, selectedCurrencySymbol}) {
+    Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, {
+        selectedCurrencyCode,
+        selectedCurrencySymbol,
+    });
+}
+
+/**
+ * Resets IOU currency to preferredCurrency
+ *
+ * @param {Object} params.selectedCurrencyCode
+ * @param {Object} params.selectedCurrencySymbol
+ */
+function resetSelectedCurrency({selectedCurrencyCode, selectedCurrencySymbol}) {
+    Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, {
+        selectedCurrencyCode,
+        selectedCurrencySymbol,
+    });
+}
```

Comment:
I am confused as to why 2 functions are required here. They perform the same function.

Reply:
Removed `resetSelectedCurrency`!